cdb2api/cdb2api.c

```diff
@@ -108,6 +108,7 @@ static void do_init_once(void)
     }
     _PID = getpid();
     _MACHINE_ID = gethostid();
+    srandom(time(0));
```

Comment:
How does this exactly help?

Reply:
without this random https://github.com/bloomberg/comdb2/blob/master/cdb2api/cdb2api.c#L1648 is called with seed 1 and it will give same value for every new run.

Reply:
random/srandom aren't threadsafe.  srandom also alters process state that other calls may depend on.   We should consider replacing them with something like

https://github.com/bloomberg/comdb2/blob/master/cdb2api/cdb2api.c#L2496

Reply:
Okay